porting/examples/freertos_nrf5_sdk/pca10040/blank/config/sdk_config.h

```diff
@@ -0,0 +1,2924 @@
+/**
+ * Copyright (c) 2017 - 2018, Nordic Semiconductor ASA
```

Comment:
ASF license?

Reply:
this is copied from nRF5 SDK thus it has Nordic's license - we can't just change license here.
this is one of reasons why this PR is not merged since we may need to figure out how to handle this properly